server/devpi_server_pytest/__init__.py

```diff
@@ -0,0 +1,7 @@
+def pytest_addoption(parser):
+    parser.addoption(
+        "--slow", action="store_true", default=False,
+        help="run slow tests involving remote services (pypi.python.org)")
+    parser.addoption(
+        "--backend", action="store",
+        help="run tests with specified dotted name backend")
```

Comment:
I've moved the plugin. I had problems before, which I thought where caused by placing it in test_devpi_server, but they were unrelated.